app/views/welcome/_next-event.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class='main'>
+  <h1> Next Event </h1>
+
+  <% if next_event %>
+    <h2><%= next_event.name %></h2>
+    <h3><%= next_event.date.strftime("%B %d, %Y")%></h3>
+
+    <% if next_event.presentations.any?%>
+      <p> Presentations from: </p>
+
+      <ul>
+        <% next_event.presentations.each do |presentation| %>
+          <li><%= "#{presentation.presenter} \u2013 #{presentation.topic}
+          (#{presentation.duration} min)" %></li>
+        <% end %>
+      </ul>
+
+    <% else %>
+        <p>There are currently no presentations for this event.
+        <%= link_to 'Sign up', new_presentation_path %>
+        to teach some people something!</p>
+    <% end %>
+
+  <% else %>
+    <p>There are no upcoming events.
+    <%= link_to 'Create one', new_event_path %>
```

Comment:
Sorry was referring to the `link_to` tag.

```
<p>There are no upcoming events.
  <%= link_to 'Create one', etc... %>
</p>
```

Your link to tag is nested inside a p tag so we would indent to show the nesting.